variables.tf

```diff
@@ -154,4 +154,10 @@ variable "vpc_id" {
   description = "Id of the VPC instance where this cluster will be provisioned"
 }
 
-#############################################################################
+variable "confirm_network_healthy" {
+  type        = bool
+  description = "Set the flag as true only if runtime is able to access kubernetes cluster otherwise change the value to false - Default : true"
```

Comment:
Update the description to:

By setting this to true, a script will run kubectl commands to verify that all worker nodes can communicate successfully with the master. If the runtime does not have access to the kube cluster to run kubectl commands, this should be set to false

Reply:
Sure. Thank you.